### ISSUE TITLE
visual feedback when click on "Send" button

### ISSUE THREAD

**Issue by jkuchar** [humhub/humhub-modules-mail#48]
When i click on poslat (send) I whould like to have visual feedback that messing is sending. Othewise I'm clicking it again and again and that I have message sent 3 times.

Simplest can be: disable button by JavaScript when user clics on it. Unblock it in 10 seconds (in case of network failure)

![image](https://cloud.githubusercontent.com/assets/133822/7726103/1cd06bee-ff00-11e4-941d-27fb8ab96116.png)

**Comment by MiniguyBrendan** [comment]
Possible duplicate of #40

**Comment by jkuchar** [comment]
Yes, it is duplicate.
